.github/workflows/main.yml

```diff
@@ -20,9 +20,9 @@ jobs:
       matrix:
         node-version: [14.x, 16.x]
```

Comment:
```suggestion
        node-version: [18.x, 20.x]
```

Reply:
Should we still use NodeJS 18 in the matrix? Our other repositories are using only version 20.

Reply:
Should we still use NodeJS 18 in the matrix? Our other repositories are using only version 20.